### PR TITLE
Update of versions and naming

### PR DESCRIPTION
**Pull request by aaitor**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -105,12 +101,12 @@
         <dependency>
             <groupId>org.java-websocket</groupId>
             <artifactId>Java-WebSocket</artifactId>
-            <version>1.3.8</version>
+            <version>1.4.0</version>
```
Comment (eruizgar91) [inline core/pom.xml]: The version number can be added as a property.

File: core/pom.xml [hunk 2]
```diff
@@ -105,12 +101,12 @@
...
         </dependency>
         <dependency>
             <groupId>io.reactivex</groupId>
             <artifactId>rxjava</artifactId>
-            <version>1.2.4</version>
+            <version>1.3.8</version>
```
Comment (eruizgar91) [inline core/pom.xml]: The same

File: core/src/main/resources/application.yml [hunk 3]
```diff
@@ -16,7 +16,8 @@ logging:
 ethereum:
   nodes:
     - name: default
-      url: https://baklava-forno.celo-testnet.org
+      #url: https://baklava-forno.celo-testnet.org
```
Comment (eruizgar91) [inline core/src/main/resources/application.yml]: Maybe we should let localhost:8545 by default, before we refactor the way of get the application.ymal

**Review by eruizgar91 — APPROVED** [review]
LGTM!

File: core/pom.xml (hunk 2 above)
Comment (aaitor) [reply core/pom.xml]: fixed

File: core/pom.xml (hunk 1 above)
Comment (aaitor) [reply core/pom.xml]: fixed

File: core/src/main/resources/application.yml (hunk 3 above)
Comment (aaitor) [reply core/src/main/resources/application.yml]: fixed

**Pull request merged**